pkg/types/types.go

```diff
@@ -67,3 +69,27 @@ func (n NodeSet) PopAny() (NodeName, bool) {
 	node, ok := n.String.PopAny()
 	return NodeName(node), ok
 }
+
+type LaunchableVersion struct {
+	ID   string            `yaml:"id"`
+	Tags map[string]string `yaml:"tags"`
+}
+
+type LaunchableStanza struct {
```

Comment:
maybe we want a `pkg/manifest` or something that both `pods` and `auth` depend on?